hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -423,8 +417,9 @@ private long calculateMaxSteps(BalancerClusterState cluster) {
             maxSteps);
       }
     }
-    LOG.info("start StochasticLoadBalancer.balancer, initCost=" + currentCost + ", functionCost="
-        + functionCost() + " computedMaxSteps: " + computedMaxSteps);
+    LOG.info("Start StochasticLoadBalancer.balancer, initial weighted average imbalance ="
```

Comment:
Start StochasticLoadBalancer.balancer, initial weighted average imbalance=0.06013745704467354, functionCost=RegionCountSkewCostFunction : (multiplier=500.0, imbalance=0.0, balanced); PrimaryRegionCountSkewCostFunction : (not needed); MoveCostFunction : (multiplier=7.0, imbalance=0.0, balanced); RackLocalityCostFunction : (multiplier=15.0, imbalance=0.0, balanced); TableSkewCostFunction : (multiplier=35.0, imbalance=1.0); RegionReplicaHostCostFunction : (not needed); RegionReplicaRackCostFunction : (not needed); ReadRequestCostFunction : (multiplier=5.0, imbalance=0.0, balanced); CPRequestCostFunction : (multiplier=5.0, imbalance=0.0, balanced); WriteRequestCostFunction : (multiplier=5.0, imbalance=0.0, balanced); MemStoreSizeCostFunction : (multiplier=5.0, imbalance=0.0, balanced); StoreFileCostFunction : (multiplier=5.0, imbalance=0.0, balanced); , computedMaxSteps=3200